rest_framework/compat.py

```diff
@@ -140,6 +140,16 @@ def value_from_object(field, obj):
     return field.value_from_object(obj)
 
 
+def getargspec(obj):  # type: tuple
+    if not hasattr(inspect, 'signature'):
```

Comment:
Return type here has been mentioned as an backward compatible for python `mypy` for python 2.
I'll remove remove it now an until there's a decision made for such thing.